Description: Fix FTBFS with Go 1.17
 As of Go 1.15, the architecture darwin/386 was deprecated, and
 it has been fully removed in Go 1.17. This was causing a failure
 in a build-time test of this package. For more information, see
 https://github.com/golang/go/issues/37610
Origin: upstream, https://github.com/sevlyar/go-daemon/commit/4575a85b8ba1ffd626da3ac85b9143300d6437eb
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/golang-github-sevlyar-go-daemon/+bug/1943761
Applied-Upstream: https://bugs.launchpad.net/ubuntu/+source/golang-github-sevlyar-go-daemon/+bug/1943761
Last-Update: 2021-09-15
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/compilation_test.go
+++ b/compilation_test.go
@@ -18,7 +18,6 @@
 	}
 
 	pairs := []string{
-		"darwin/386",
 		"darwin/amd64",
 		"dragonfly/amd64",
 		"freebsd/386",
